payment/src/test/java/org/killbill/billing/payment/api/TestPaymentApi.java

```diff
@@ -1446,7 +1446,7 @@ public void testVerifyJanitorFromPendingDuringCompletionFlow() throws PaymentApi
             final Payment completedPayment = createPayment(TransactionType.AUTHORIZE, initialPayment.getId(), initialPayment.getExternalKey(), transactionExternalKey, authAmount, PaymentPluginStatus.PROCESSED);
             Assert.fail();
         } catch (final PaymentApiException e) {
-            Assert.assertEquals(e.getCode(), ErrorCode.PAYMENT_INVALID_OPERATION.getCode());
+            Assert.assertEquals(e.getCode(), ErrorCode.PAYMENT_ACTIVE_TRANSACTION_KEY_EXISTS.getCode());
```

Comment:
> The authorize duplication will fail because of linked state machine has no entry from Auth_success to another authorize machine.

To double check my understanding: it used to go through for capture (and other operations) because we support partial captures (link from `CAPTURE_SUCCESS` to `CAPTURE_INIT` and the state machine doesn't enforce the unicity of keys)?